saleor/static/dashboard-next/components/ExtendedPageHeader/ExtendedPageHeader.tsx

```diff
@@ -41,34 +42,35 @@ const styles = (theme: Theme) =>
   });
 
 interface ExtendedPageHeaderProps extends WithStyles<typeof styles> {
+  back?: boolean;
   children?: React.ReactNode;
   className?: string;
   title?: React.ReactNode;
-  onBack?();
 }
 
 const ExtendedPageHeader = withStyles(styles, { name: "ExtendedPageHeader" })(
-  ({
-    children,
-    classes,
-    className,
-    onBack,
-    title
-  }: ExtendedPageHeaderProps) => (
+  ({ back, children, classes, className, title }: ExtendedPageHeaderProps) => (
     <div className={classNames(classes.root, className)}>
-      {onBack && (
-        <IconButton
-          color="inherit"
-          className={classes.menuButton}
-          onClick={onBack}
-        >
-          <ArrowBackIcon />
-        </IconButton>
+      {back && (
+        <Back>
+          {back => (
+            <IconButton
+              color="inherit"
+              className={classes.menuButton}
+              onClick={back}
+            >
+              <ArrowBackIcon />
+            </IconButton>
+          )}
+        </Back>
       )}
       {title}
       <div className={classes.action}>{children}</div>
     </div>
   )
 );
 ExtendedPageHeader.displayName = "ExtendedPageHeader";
+ExtendedPageHeader.defaultProps = {
+  back: false
```

Comment:
Q: Won't be better to set the default value to true? I see many places when you pass it as so.
If not this line is really useless since `undefined` is falsy.